packages/web-wallet/src/components/SendDialog.tsx

```diff
@@ -63,10 +64,11 @@ const createSendFormSchema = (network: string) =>
           });
         }
       }),
-    timelock: z.date().optional().refine(
-      (date) => !date || isFutureDate(date),
-      'Timelock date must be in the future'
-    ),
+    // TODO: Re-enable when snap supports timelock
+    // timelock: z.date().optional().refine(
+    //   (date) => !date || isFutureDate(date),
+    //   'Timelock date must be in the future'
+    // ),
```

Comment:
timelock on sent transactions?
I think it does, I just sent a transaction with timelock today using the web-wallet